libpod/image/pull.go

```diff
@@ -149,6 +149,13 @@ func (ir *Runtime) pullGoalFromImageReference(ctx context.Context, srcRef types.
 		// Need to load in all the repo tags from the manifest
 		res := []pullRefPair{}
 		for _, dst := range manifest[0].RepoTags {
+			//check if image exists and gives a warning of untagging
+			localImage, err := ir.NewFromLocal(dst)
+			imageID := strings.TrimSuffix(manifest[0].Config, ".json")
+			if err == nil && imageID != localImage.ID() {
+				logrus.Errorf("the image %s already exists, renaming the old one with ID %s to empty string", dst, localImage.ID())
```

Comment:
Suggest rewording - `"tag %s already exists, untagging but not deleting old image"

Reply:
Oh, but you'd want to keep the ID in there... maybe `tag %s already exists as ID %s, untagging but not deleting old image`?

Reply:
untagging would mean removing the tag, maybe I could reword it to 
`tag %s already exists as ID %s, tagging the old image as an empty string`

Reply:
Back to my original point, unless I'm being oblivious, this code is not actually tagging the old image as an empty string is it?  I think in this message you need to tell the user what action they need to do as we're not doing it for them.

Reply:
We're specifically retagging as empty string? I think internally that's considered an untag of the old image... @baude that sound right to you?

Reply:
Internally this set's the name of the image  `image.Names = stringSliceWithoutValue(image.Names, name)` in the file https://github.com/containers/storage/blob/master/images.go .

The thing that is confusing me is that if I do `podman rmi imagename` then if the image exists with another tag then it simply gets untagged and is not visible when I run `podman images`, but in this case, the image is visible in `podman images` as `<none>` .

Reply:
...Could this be the dreaded literal <none> tag returning... Let me poke around